lib/pull-manager.js

```diff
@@ -8,6 +8,12 @@ var sockets = [];
 var pulls = [];
 
 var pullManager = module.exports = {
+   getOldestAllowedPullTimestamp: function() {
```

Comment:
Can we rename this function to `getOldestAllowedPullTimestampInSeconds` ? I see we are using it to represent Unix timestamps but we are working with both `seconds` and `milliseconds` in other parts of the code which was making it hard to decipher which format we are trying to use.

Reply:
I understand the confusion. Maybe better to have this return a `Date` object and the caller can do what it needs.

Reply:
Done!